tempesta_fw/server.h

```diff
@@ -42,12 +42,14 @@ typedef struct tfw_scheduler_t TfwScheduler;
  *
  * @list	- member pointer in the list of servers of a server group;
  * @sg		- back-reference to the server group;
+ * @sched_data	- private scheduler data for the server;
  * @apm		- opaque handle for APM stats;
  */
 typedef struct {
 	TFW_PEER_COMMON;
 	struct list_head	list;
 	TfwSrvGroup		*sg;
+	void			*sched_data;
```

Comment:
We use the same member name `sched_data` in `TfwSrvGroup`. Both have the same meaning and usage, so i think it is a good idea to have the same names for them. Should i rename both to `schref`?